### DISCUSSION TITLE
How to enable live preview of files for `consult-find`?

### DISCUSSION THREAD

**Discussion by IceAsteroid** [minad/consult#1108] [Q&A] [answered]
Is there a way to enable file preview for `consult-find`? I found that commands like `consult-grep` support file preview, but not for this command.

Furthermore, if it's possible, can the live preview be bound with a keybinding instead of being automatically executed like customizing other `consult-*` commands that support live preview?

I've read the **Live previews** section of the readme page.

Here's the quote about customizing the live-preview to be triggered by keybindings.

> A safe recommendation is to leave automatic immediate previews enabled in general and disable the automatic preview only for commands where the preview may be expensive due to file loading. Internally, Consult uses the value of this-command to determine the :preview-key customized. This means that if you wrap a consult-* command within your own function or command, you will also need to add the name of your custom command to the consult-customize call in order for it to be considered.

**Comment by minad** [comment, marked as answer]
You have to configure the `:state` argument via `consult-customize`, i.e., `:state (consult--jump-state)`. Maybe also search the discussion forum, the issue tracker and the wiki. This has certainly be asked before. Maybe you could extend the wiki please in case such an example is missing there?

**Reply by IceAsteroid** [reply]
Thanks for the clue, I'll try to configure it, and extend the wiki when finished.

**Reply by IceAsteroid** [reply]
I think I'll make the preview bound to a key in vertico, since not all files are suitable to be automatically previewed.
